pkg/webhook/v1beta1/pod/inject_webhook.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"fmt"
+	trialcontroller "github.com/kubeflow/katib/pkg/controller.v1beta1/trial"
```

Comment:
Please move the package to the next group. Maybe after https://github.com/kubeflow/katib/pull/1560/files#diff-375c6a13fa9e97139720677b5362b2e5b880687cbef618da72701bce31553d35R39